### DISCUSSION TITLE
Is the Resolution too high?

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#257] [Q&A] [answered]
One of my cameras started to fail so I replaced it with one that has a lot higher resolution (2688x1520). The problem is that the detection video starts at the timestamp of **39:57** but the detection image is  for the timestamp of **39:48**. The image shows what was detected but by the time the video started recording the object was long gone. Tried increasing the `lookback `setting to 12 but it didn't change anything.

**Comment by roflcoopter** [comment]
Could be that the detector is too slow which causes a delay in the detection, will look into discarding old frames.
Moving this to an issue instead

**Reply by gniknalu** [reply]
OK - thank you!

**Comment by gniknalu** [comment]
Figured out what is going on. I was having issues with items detected on one camera showing up on ALL cameras. Spent literally a whole day trying different configurations and stumbled upon `roflcoopter/viseron:dev` which DID NOT have this issue. Little did I know that it didn't have the issue because it wasn't using my GPU! All 8 of those i7 CPU's were at 80% or more but it has been working - with lag - for the past couple of weeks. It was only when I decided to split up the load with a second i7 box that I realized - thanks to your assistance - that it wasn't using the GPU because I took the configuration from the first box and copied it to the second. 

So now I'm back to the original issue of detections from one camera being on all the other cameras. BUT at least I'm using the GPU and the 8 CPU's are back to ~15% utilization.

Closing this one.

**Reply by roflcoopter** [reply]
Does this duplication happen when you run with less cameras? Is it more prevalent with more cameras?

**Comment by gniknalu** [comment, marked as answer]
Fixed!
